crossdock/client/tchclient/thrift.go

```diff
@@ -21,11 +21,16 @@
 package tchclient
 
 import (
+	"reflect"
+	"strings"
 	"time"
 
 	"github.com/yarpc/yarpc-go/crossdock-go"
+	"github.com/yarpc/yarpc-go/crossdock/client/gauntlet"
+	"github.com/yarpc/yarpc-go/crossdock/client/pointer"
 	"github.com/yarpc/yarpc-go/crossdock/client/random"
 	"github.com/yarpc/yarpc-go/crossdock/thrift/gen-go/echo"
+	"github.com/yarpc/yarpc-go/crossdock/thrift/gen-go/gauntlet_apache"
```

Comment:
No separators are preferred in general. If a separator is needed, underscores have the advantage that they can be part of the package name, so the base name and the package name match. Hyphens cannot be part of the package name so if they're used in the base name, there must be some obvious convention mapping the base name to the package name (yarpc-go => yarpc, thriftrw-go => thriftrw, foo-bar => foo_bar).

See also https://blog.golang.org/package-names